### ISSUE TITLE
[Bug]: Build (Run gradlew | Install pods) error if using with `braze-react-native-sdk` (>= 2.0.0)

### ISSUE THREAD

**Issue by karengrigoryan** [braze-inc/braze-expo-plugin#12]
### Which Platforms?

Both

### Which React Native Version?

0.70.5

### Which @braze/expo-plugin version?

0.6.0

### Which react-native-appboy-sdk SDK version?

2.1.0

### Repro Rate

100%

### Steps To Reproduce

1. Install `@braze/expo-plugin`
2. Install `@braze/react-native-sdk` (instead of deprecated `react-native-appboy-sdk`)
3. Build/prebuild the app

### Expected Behavior

It should correctly build the app.

### Actual Incorrect Behavior

It throws an error for both platforms:

Logs from EAS.

### Android

```
[stderr] A problem occurred evaluating project ':braze-expo-plugin'.
[stderr] > Project with path ':react-native-appboy-sdk' could not be found in project ':braze-expo-plugin'.
```

### iOS

```
[!] Unable to find a specification for `react-native-appboy-sdk` depended upon by `ExpoAdapterBraze`
```

### Verbose Logs

```shell
> Task :react-native-gradle-plugin:compileJava NO-SOURCE
> Task :react-native-gradle-plugin:classes
> Task :react-native-gradle-plugin:inspectClassesForKotlinIC
> Task :react-native-gradle-plugin:jar
[stderr] FAILURE: Build failed with an exception.
[stderr] * Where:
[stderr] Build file '/home/expo/workingdir/build/node_modules/@braze/expo-plugin/android/build.gradle' line: 48
[stderr] * What went wrong:
[stderr] A problem occurred evaluating project ':braze-expo-plugin'.
[stderr] > Project with path ':react-native-appboy-sdk' could not be found in project ':braze-expo-plugin'.
[stderr] * Try:
[stderr] > Run with --stacktrace option to get the stack trace.
[stderr] > Run with --info or --debug option to get more log output.
[stderr] > Run with --scan to get full insights.
[stderr] * Get more help at https://help.gradle.org
[stderr] BUILD FAILED in 1m 46s
```

### Additional Information

It seems like `@braze/expo-plugin` is not compatible with the new `@braze/react-native-sdk` package. It works correctly with the deprecated`react-native-appboy-sdk`.

**Comment by wesleyorbin** [comment]
Hi @karengrigoryan. We are working on an update to the Expo plugin to make it compatible with the newest version of the React Native SDK. It should be released by the end of this week. I'll update you here when it's available.

**Comment by karengrigoryan** [comment]
@wesleyorbin thanks a lot.

A small note: the current [documentation](https://www.braze.com/docs/developer_guide/platform_integration_guides/react_native/react_sdk_setup/#step-1-integrate-the-braze-library) for RN/Expo uses `@braze/react-native-sdk` (at the time of writing this comment).

**Comment by wesleyorbin** [comment]
Hi @karengrigoryan. We just released v1.0.0 of the plugin, which contains support for React Native SDK 2.0.2+. Apologies for the confusion with the documentation.
